xline/src/storage/lease_store/mod.rs

```diff
@@ -58,7 +58,14 @@ where
 
 /// Collection of lease related data
 #[derive(Debug)]
-struct LeaseCollection {
+pub(crate) struct LeaseCollection {
```

Comment:
I recommend extracting the `LeaseCollection` into an independent file. In doing so, we can prevent the other stores from re-compiling if we change the `LeaseStoreBackend` code.

Reply:
Fixed